src/server/server.ts

```diff
@@ -18,7 +18,7 @@ async function startServer(): Promise<void> {
   server.register(mercurius, {
     schema,
     resolvers,
-    graphiql: 'playground',
+    graphiql: true,
```

Comment:
Apparently a custom path is no longer possible.

Reply:
Or should we use our own, 'persistent' path instead (e.g. stick with `/playground`)? If Mercurius decides to use another GraphQL editor again in the future, shipping with its own path, we too have to update our paths.

Reply:
Apparently a custom path is no longer possible (`/playground` or `playground` didn’t work). But please note we’re still redirecting `/` to the playground. So as long as we refer developers to https://termennetwerk-api.netwerkdigitaalerfgoed.nl rather than https://termennetwerk-api.netwerkdigitaalerfgoed.nl/playground we’re fine.

Reply:
Thanks! I was thinking about the use case where users have bookmarked `/playground` already. Will this change return a 404 when requesting this path? If so, would it be an option to not only redirect `/` to the playground, but also `/playground` (for backward compatibility)?

Reply:
Added a redirect!